store/tikv/region_cache.go

```diff
@@ -66,17 +79,61 @@ func (c *RPCContext) GetStoreID() uint64 {
 	return 0
 }
 
+func (c *CachedRegion) isValid() bool {
+	lastAccess := atomic.LoadInt64(&c.lastAccess) // we may not need an atomic load, I think a simple load is enough?
```

Comment:
We need to use atomic, or the race detector will complain. Please remove the comment.

Reply:
OK, Thanks~

Reply:
OK, Thanks~
By the way, in fact, we do not need to use atomic.